webapp/components/Comment.vue

```diff
@@ -132,6 +135,9 @@ export default {
     editCommentMenu(showMenu) {
       this.openEditCommentMenu = showMenu
     },
+    updateComment(comment) {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-11T15:21:08Z" title="Wednesday, September 11th 2019, 5:21:08 pm +02:00">Sep 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -132,6 +135,9 @@ export default {
     editCommentMenu(showMenu) {
       this.openEditCommentMenu = showMenu
     },
+    async updateComment(comment) {
```


why is this `async` if there is no `await`?
Is this really the best way to handle this situation @Tirokk @roschaefer 
we are emitting an event from a grandchild to a parent component, which just emits an event to its parent(granparent of CommentForm) so that it can update the list??
Seems strange this would be the best way to do this....

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-12T07:20:08Z" title="Thursday, September 12th 2019, 9:20:08 am +02:00">Sep 12, 2019</time>_

---


The `async` slipped in by copy and paste I suppose and I have overseen it. @mattwr18 
I removed it.

This @roschaefer suggested and I followed him, because I don’t know a better way yet. And we need something what works for sure for the moment.

May the `vue-states` + `vue-history` packages from our last Vue MeetUp would help here in the future?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-12T16:35:44Z" title="Thursday, September 12th 2019, 6:35:44 pm +02:00">Sep 12, 2019</time>_

---


maybe... that would be cool :smile: